### PR TITLE
v0.56.0 — Added badge component and updates cards component.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Added
- Optional `badge` component.

### Changed
- `card` component modifiers.
- Applied relative position to `card` component.

### CODE REVIEW

File: src/scss/components/_badges.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Badge Component
+ * =================================
+ * Example – Menu – Restaurant delivery information
+ *
+ * Used for styling small pieces of information.
+ *
+ * Documentation:
+ * https://fozzie.just-eat.com/styleguide/ui-components/custom/badges
```
Comment (DamianMullins) [inline src/scss/components/_badges.scss]: I'll create this once this PR has been merged.

File: src/scss/components/_badges.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ */
+
+$badge-bgColor    : $grey--offWhite !default;
```
Comment (ashleynolan) [inline src/scss/components/_badges.scss]: Super NIT – this looks slightly off the rest of the indenting below 😞

Comment (DamianMullins) [reply src/scss/components/_badges.scss]: Fixed.

**Comment by bulutcy** [comment]
What is cards components used for? Is it for listing credit cards, or just a generic component to render block pieces?

**Comment by DamianMullins** [comment]
@bulutcy It's a generic component which wraps content, [you can see it in use on the docs website](https://fozzie.just-eat.com/styleguide/ui-components/custom/star-ratings); it's the white boxes wrapping the sidebar and main content area.

**Pull request merged**